production.env

```diff
@@ -68,13 +59,13 @@ FTPS_PASSIVE_PORTS=8100-8399
 
 # Various helpers
 ADMIN_EMAIL="MiG Info <mig@migrid.test>"
-ADMIN_LIST=
-SMTP_SENDER=
+ADMIN_LIST="/C=DK/ST=NA/L=NA/O=EX/OU=NA/CN=John Doe/emailAddress=jd@example.com"
+SMTP_SENDER="Do Not Reply <no-reply@example.com>"
 LOG_LEVEL=info
 TITLE="Minimum intrusion Grid"
 SHORT_TITLE=MiG
 MIG_OID_TITLE="MiG"
-EXT_OID_TITLE="External"
+EXT_OID_TITLE="External OpenID"
```

Comment:
AFAIK the OpenID part is already implicitly added where used in the code, so 'External' should do.